spec/mongoid/touchable_spec.rb

```diff
@@ -2,14 +2,31 @@
 
 require "spec_helper"
 
-describe Mongoid::Association::Touchable do
+describe Mongoid::Touchable do
 
   describe "#touch" do
 
+    context "when the document has no relations" do
```

Comment:
This test invokes `touch` on the document. Whether the document has any *associations* (not relations) doesn't matter.

Reply:
I disagree that it doesn't matter; the entire purpose of this ticket is to add the ability to call `touch` on documents without associations, and that's what this is testing.